### ISSUE TITLE
Suggestion - Elevate ALL businesses | Reward highest RP potential

### ISSUE THREAD

**Issue by navonody1** [InZidiuZ/op-framework-issue-tracker#3269]
### Assurance

I have read the `README.md`

### Summary

Add to businesses the ability to clean (like prison script). This eliminates the need to implement a clock in system and allows employees that work at a business to have an incentive to be there and not just stand around.

- *Create multiplier that increases the frequency of the potential action events that activates based on how many people are 	around.

- *Create multiplier that increases the frequency of potential action events based on consecutive time working (example: 	increase every 60 minutes).

	***Multipliers reset if you stay away from the business radius too long.***

**Potential Action Events:**
Things like: Wipe down counters (tables & chairs), dust off TVs, clean up oil spills, mop/sweep the floor, clean bathrooms, etc.
Actions show up after a certain amount of time working.

- *Alerts that an action event is available could appear in a black box at the top left, similar to when you hover over a garage:
![garage prompt](https://github.com/InZidiuZ/op-framework-issue-tracker/assets/96219411/e62900a6-9c38-45b0-9f01-3f0a32b23086)
in combination with the task indicator arrow (like the non-whitelisted jobs use)

**Potential Loot [random % chance]:**

- _All - Cash drops_
- _Mechanic Shops_ - 10mm socket wrench [rare], loose car parts, loose bolts, hammers, pipe wrenches

	*Loose car parts - sold where 2nd Gear used to be (it's now a scrap yard)
	*Loose bolts - sold at 2nd Gear location
	*10mm socket wrench - sold at 2nd Gear location

- _Restaurants_ - Wallets (with bonds in them), purses (with bonds in them), silver watches, gold watches, necklaces

	*Bonds - can be sold/traded in at any bank
	*Allow wallets to be sold at Bazaar
	*Allow purses to be sold at Bazaar

- _Bars_ - Lost rings [variety], key ring full of keys, silver watches, gold watches, necklaces

	*Key ring of keys - can be sold at Bazaar

### Reason

- When looking at the totality of earning potential, whitelisted jobs should be the highest paying activities in the server. 

- The activities that have the highest RP potential, should have the highest reward.  

- As it stands now, we've created a culture where someone shows up to work for 10-15 minutes or simply answers a Twitter post, then leaves immediately. 

- Alleviate negative interactions where it can feel like co-workers are competing with each other. 

**TLDR;**
In my opinion, there's a balance between RP and valuing people's time investment. I think legal business jobs should be a far better alternative to the hunting, mining, diving grind set where there is little to no RP potential. This suggestion creates more RP scenarios AND incentive for workers to WANT to be in their respective businesses across the city.

**Comment by SluggsV2** [comment]
I do agree this would be cool! For me being in realestate, would be cool if we had to trim bushes or something in that general thing,

for loot I wouldn’t really know what to suggest, looking st the suggested above maybe the, Key ring, Rings, Pet Toys, 

or for a lawyer maybe we can have a little inside mini game in the court house, would be a simple game, 

Lawyers would use a tablet like /comp for animation, (would be a new tablet “Attorneys Tablet” and inside it would give local cival cases that you have to find them guilty or non guilty, or something along those lines,

**Comment by navonody1** [comment]
> In all honesty, whenever i noclip around, I see mechanic shops barely keeping up with the number of repairs needing to be done. Back in the day i used to sit at PDM for 4 hours a day. If your employees are not at their place of work when there is demand for the business, then they should consider a different place of work that they maybe enjoy more? And if your business is not getting any customers, then you don't need incentive to keep the workers there, you need to rethink your business strategy or talk to state to see about potential funding to help out.
> 
> Business RP is more than just "standing around waiting for customers". You can do events, hand out bonuses, get people engaged! If people aren't buying the items your business sells because you put a huge markup on your items, then nobody will buy them. (Zombie pills for example are a waste of money once they reach a certain price, causing people to not want to buy them or look for other sources like trash. This is called "[competition](https://en.wikipedia.org/wiki/Competition_(economics))", but we are getting sidetracked.) The issue you are pointing out with this issue real, don't get me wrong, but it's an issue with the community and the fact people love grinding pixel money. I'm down to help out by adding unique items (like the new kinder surprise egg for burger-shot & bean for example) to draw people in there and there are many more planned, but simply giving employees more money for standing around it not the solution.
> 
> We barely had a fraction of the things to do that we do now back 2-3 years ago and it worked perfectly fine. Sometimes better than now even, I'd say.

I understand what you mean, but the BEST incentive is money if it's true what you say about how people love grinding pixel money. This was an attempt to shift the grind from the newer activities that are solo RP based, to the businesses. Right now, someone that has 3 hours to "RP" is met with a decision: Do I go stand at a business with little to no engagement? Or do I use this time to go grind what I can to cap? <-- one of these choices, at the very least, makes the time spent worth it. I agree a culture change needs to happen, but that's not going to happen in the current state without incentive.

There is a fear that this suggestion, if added, would just end with people grinding as well, however, a cap could be placed in the same way the other "grinding" activities are capped.  

Calling this suggestion stupid was a bit harsh. This was just a rough draft of an idea, but since it's such a hard no...back to the drawing board I guess.

**Comment by navonody1** [comment]
Also, in the past I've suggested adding things like tokens or tickets to the Arcade that could be redeemable for prizes (new exclusive shoulder pets, figurines, etc.) and that was shot down just as harshly as this suggestion. It's disheartening to make a suggestion that could benefit the Arcade, but everyone looks at the Arcade as too powerful and writes it off.

**Comment by ristwell** [comment]
> No, this is stupid. The businesses you mentioned already have things that make money. That's their whole purpose. Mechanics make good money doing repairs, restaurants make money selling food. Adding an NPC tasks for people to do in every business will result in people ONLY doing that task, grinding it as much as possible, it's precisely what happens, EVERY TIME. This will lead people away from interacting with others, there are plenty of other ways to make extra cash (and others that are planned) that, if done right, make you very good money pretty quickly.

“too much work, not planned, suggestion stupid” If I had a nickle…

Brain dead response imho

**Comment by coalaura** [comment]
> It sounds to me like you are trying to hold Arcade accountable for something we can not control and thereby contributing to us being punished for the prices that the manufacturer ultimately forced our hand to set

Holding nobody accountable and punishing nobody. This issue was directed to all businesses including restaurants, mechanic shops and bars. If this is an issue the arcade is having then opening an issue to overhaul every business in the city is not really the right thing. Things like store prices can be adjusted, you just have to actually let us know. I've reduced the arcades z-pill price down to 1k flat to aid with the sales :)

**Comment by navonody1** [comment]
> > It sounds to me like you are trying to hold Arcade accountable for something we can not control and thereby contributing to us being punished for the prices that the manufacturer ultimately forced our hand to set
> 
> Holding nobody accountable and punishing nobody. This issue was directed to all businesses including restaurants, mechanic shops and bars. If this is an issue the arcade is having then opening an issue to overhaul every business in the city is not really the right thing. Things like store prices can be adjusted, you just have to actually let us know. I've reduced the arcades z-pill price down to 1k flat to aid with the sales :)

Can the cost of z pills be lowered to $935? + 7% tax makes it a flat $1k for people behind counter. I asked for that back in #2770.

**Comment by coalaura** [comment]
> “too much work, not planned, suggestion stupid” If I had a nickle…
> 
> Brain dead response imho

Apologies for saying this issue is *stupid*. It is very detailed and thought out, yet had flaws that I pointed out in my later comments, going more into detail WHY I decided to close the issue. Not everything that lands here on github is perfect and can be implemented. The OP framework powers 23 different servers (at this point in time). Not everything can be done exactly the way people imagine

**Comment by ristwell** [comment]
> > “too much work, not planned, suggestion stupid” If I had a nickle…
> > Brain dead response imho
> 
> @ristwell Apologies for saying this issue is _stupid_. It is very detailed and thought out, yet had flaws that I pointed out in my later comments, going more into detail WHY I decided to close the issue. Not everything that lands here on github is perfect and can be implemented. The OP framework powers 23 different servers (at this point in time). Not everything can be done exactly the way people imagine

My point is that the “No this is stupid” came before any reasoning. I disagree with your reasoning and that’s fine people can have a difference of opinion. But one such as yourself in a position of power coming at suggestions in a manner of “no this sucks not adding this because it bad” instead of engaging in discourse is the problem. There is no discussion. You said this is stupid i’m not doing it listed some reasons that are addressed already in the post and close the suggestion. TO ME it seems there isn’t a genuine attempt to look at suggestions in a constructive manner at all. Only destructive. You didn’t say oh well it can be abused, so what would be the fix for that? Even thought it can’t because as he said the time and frequency is low and the pay out is low it’s just incentive enough to keep someone hanging around the business. You aren’t asking questions, you are making conclusions that something won’t work. If you think there’s an issue with people not hanging around  business, and you’re fix is bonus’ and state payouts? I think that’s a far worse idea than just small cashable incentive to hang around the business. I’ve done days at the arcade standing there for upwards of 5-6 hours with little to no customers and no rp and just sitting there tweeting our and playing pokemon on my switch waiting for something to happen. Sometimes i’m there for like 8 minutes and someone wants to buy a bunch of stuff and I make 30k in one transaction. The point is that it’s inconsistent. 

Whatever enough of me ranting. I think your attitude on how you approach suggestions is negative only and needs to change and I think this suggestion is well thought out, cool, value and RP add. You disagree and I disagree with your reasoning.

**Comment by coalaura** [comment]
@ristwell i'm not an employee of a corporate business. I am free to express my opinion on something even if that includes the word stupid. I get an email for every response to github issues and i read every comment. If someone asks for more clarification i can provide it. I don't like spending a lot of time writing up these ginormous explanations if i dont have to. Does that make me look like i dont care? maybe, but i'd rather spend my time doing something that i enjoy (developing the framework) than arguing with people on the internet

**Comment by ristwell** [comment]
> @ristwell i'm not an employee of a corporate business. I am free to express my opinion on something even if that includes the word stupid. I get an email for every response to github issues and i read every comment. If someone asks for more clarification i can provide it. I don't like spending a lot of time writing up these ginormous explanations if i dont have to. Does that make me look like i dont care? maybe, but i'd rather spend my time doing something that i enjoy (developing the framework) than arguing with people on the internet

If you're going to speak like that to people you have to understand that people will respond in kind. If you can't take the criticism of your criticism, don't speak like that to people. And this is discourse, not an argument.

**Comment by ristwell** [comment]
> > @ristwell i'm not an employee of a corporate business. I am free to express my opinion on something even if that includes the word stupid. I get an email for every response to github issues and i read every comment. If someone asks for more clarification i can provide it. I don't like spending a lot of time writing up these ginormous explanations if i dont have to. Does that make me look like i dont care? maybe, but i'd rather spend my time doing something that i enjoy (developing the framework) than arguing with people on the internet
> 
> If you're going to speak like that to people you have to understand that people will respond in kind. If you can't take the criticism of your criticism, don't speak like that to people. And this is discourse, not an argument. 

side note: if you hadn't opened with "No this is stupid" I wouldn't have replied, I'd be bummed that we couldn't get a cool thing. Just so you understand how you as a decision maker on this particular platform, have more weight to your words than someone who doesn't get to make decisions.

**Comment by GodyFromDiscord** [comment]
It's a great idea but nothing much over a command usually gets implemented... 

Such great potential here as stated by others

**Comment by Breezy368** [comment]
+1 for something for employees to do while the owner drums up business. My guys just standing there listneing to music is boring and we're playing a game. Give em something to do to keep them around the busines and make a little extra cash imo. You talk like business owners arent out there trying to hustle for their businesses currently. Ive driven a bus around the entire city offering to drive people to paleto bar and no one wants to go, they want to make money too. WOuld be nice that spending 50k actually has some type of return, its why the city RP is dead right now, imo. Theres no accumulating places that permit RP to happen.

I think if you spend some time watching attolicas you'd see me sitting in the bar a lot, but also out handing out business cards, tweeting, druming up business for the bar and getting pretty minimal to no returns out of it both RP and financially. Would be nice if you'd consider there's a change in the community and we'd like things to do as business owners who spend 4 hours at a bar alone hoping someone comes by after spending 12 hours earlier in the week telling everyone it will be open....
